packages/documentation/src/stories/components/badge/badge.stories.ts

```diff
@@ -126,7 +126,7 @@ function externalControl(story: any, { args }: StoryContext) {
 
 // RENDERER
 function getDefaultContent(args: Args) {
-  if (!args.nestedBadge) return html`${args.text}`;
+  if (!args.nestedBadge) return html`<span>${args.text}</span>`;
```

Comment:
Do we still need that?
I would suggest to implement the if statement within the return value below.
Something like

```jsx
return html`
  <span>${html`${args.text}`}</span>
  ${args.nestedBadge ? <span class="badge">10</span> : nothing}
`;
```